### ISSUE TITLE
Issues finding modules from EJS using electron packager

### ISSUE THREAD

**Issue by iamterryclark** [electron-userland/electron-json-storage#171]
I am trying to package my application using electron-packager and have required the EJS module into my electron `main.js` file.

```
// main.js
const storage = require('electron-json-storage');

// Do other electron stuff and store some data
```

I added EJS in my dependencies but am not able to run the packaged application as it complains about not finding `lodash` ... and I presume all of the other dependencies located in EJS `package.json`

I added EJS as a dependency of my application so that my `main.js` would find the module. Otherwise electron would prune this from the resources folder when packaging.

```
 "dependencies": {
    "electron-json-storage": "^4.5.0"
  },
```

Anyone else having this issue?

**Comment by jviotti** [comment]
Hey @iamterryclark ,

Are you sure this is related to `electron-json-storage`? Does the issue disappear if you uninstall `electron-json-storage`? If not, it might be a problem to report to `electron-packager` instead.

**Comment by iamterryclark** [comment]
Thanks for getting back.

This is the only external module I am using on the main process side of the code. Other than with `child` and `path` modules along with `require('electron-devtools-installer')` which only gets required in dev mode. I have packaged the app many times before adding this module with no issues.

I will keep looking into packager, in case I have `--ignored` something crucial.

**Comment by iamterryclark** [comment]
My apologies,

I am also developing this app inside a monorepo, which hoists dependencies up. more information on this I found here https://github.com/lerna/lerna/blob/main/doc/hoist.md. No solution as yet, i need to stop the hoisting on this modules dependencies and their sub dependences.

must be an thing with requiring modules. I guess a work around would be for me to use this on the renderer process code instead,

Closing issue
